dandiapi/api/management/commands/migrate_published_version_metadata.py

```diff
@@ -0,0 +1,56 @@
+from difflib import ndiff
+from pprint import pformat
+
+from dandischema import migrate
+from django.db.models import Q
+import djclick as click
+
+from dandiapi.api.models import Version
+from dandiapi.api.tasks import write_manifest_files
+
+
+@click.command()
+@click.argument('dandiset')
+@click.argument('published_version')
+@click.argument('to_version')
+def migrate_published_version_metadata(dandiset: str, published_version: str, to_version: str):
+    click.echo(
+        f'Migrating published version {dandiset}/{published_version} metadata to version {to_version}'  # noqa: E501
+    )
+    version = Version.objects.filter(~Q(version='draft')).get(
+        dandiset=dandiset, version=published_version
+    )
+    metadata = version.metadata
+
+    # If there is no schemaVersion, assume the most recent
```

Comment:
The code doesn't match up with this comment--`to_version` is whatever version the caller requested, and not necessarily the most recent version, right?